boards/shields/adafruit_ina219/adafruit_ina219.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025 Jonas Berg
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&zephyr_i2c {
+	status = "okay";
+
+	/* The shunt resistor on the board is 0.1 Ohm.
+	 * The default max shunt voltage is +/-320mV,
+	 * resulting in a max current range of +/-3.2 Amps.
+	 * Calculate the current LSB value as 3.2/2^15 A = 122 uA.
+	 * Round the current LSB value upwards to 200 uA.
+	 */
+	adafruit_ina219: ina219@40 {
+		status = "okay";
+		compatible = "ti,ina219";
+		reg = <0x40>;
+		shunt-milliohm = <100>;
+		lsb-microamp = <200>;
```

Comment:
I do not understand why `122` is not used here instead of round up?

Reply:
That is how it is done in the INA219 datasheet: https://www.ti.com/lit/ds/symlink/ina219.pdf See section 8.5.1 "While this value yields the highest resolution, it is common to select a value for
the Current_LSB to the nearest round number above this value to simplify the conversion of the Current Register
(04h) and Power Register (03h) to amperes and watts respectively."

Of course I can change the value, if that is the consensus here.